.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -8,32 +8,99 @@ assignees: ''
 
 # Onboarding
 
-By creating this ticket, you are requesting to be onboarded as a developer on the Azul team.
-The action items in this ticket will be carried out by you and existing team
-members.
+By creating this ticket, you are requesting developer onboarding to team 
```

Comment:
```suggestion
By creating this ticket, you are requesting to be on-boarded to team 
```

The ticket should be created by the supervisor of the employee. Please reword accordingly. Any initial actions should be reflected in the checklist rather then as prose preceding the checklist.

Additionally, I want to require that employees create GitHub accounts exclusively for use for work, and that their GitHub account names are `{ucsc_account_name}-ucsc`. Please work that in as well. Account for the case that an employee already has such an account.